src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+import './App.css';
+
+function App() {
+  const [greetings, setGreetings] = useState([]);
+
+  useEffect(() => {
+    fetch('/api/v1/greetings')
+      .then(response => response.json())
+      .then(data => {
+        console.log('Data from API:', data); // Log the data
+        setGreetings(data);
+      })
+      .catch(error => console.error('Error fetching greetings:', error));
+  }, []);
+
+  return (
+    <div>
+      <h1>Greetings App</h1>
+      <ul>
+        {greetings.map(greeting => (
+          <li key={greeting.id}>{greeting.greeting}</li>
+        ))}
+      </ul>
+    </div>
```

Comment:
- [ ] Kindly use redux to get the message from the API endpoint as requested in the project requirement.
![Screenshot from 2024-02-03 00-33-48](https://github.com/gilberthappi/hello-react-front-end/assets/91142432/2351c25e-134d-4516-8ef1-c28dde87dd0c)